src/common/xo/index.js

```
@@ -972,12 +972,19 @@ export const addAcl = ({subject, object, action}) => (
   call('acl.add', resolveIds({subject, object, action}))::tap(
     subscribeAcls.forceRefresh
   )
-)
+).catch(err => error('Add ACL', err.message || String(err)))
```

Never swallow the error.